demo/common/assets.js

```diff
@@ -1322,23 +1322,74 @@ shakaAssets.testAssets = [
   // }}}
 
   // MPEG-5 LCEVC assets {{{
-  /* LCEVC Enabled Content with LCEVC Encoded Stream */
+  /* LCEVC Enhanced Contents */
```

Comment:
@v-nova-vinod In order to simplicate this PR, I moved the assets changes to another PR. https://github.com/shaka-project/shaka-player/pull/5522

Reply:
@v-nova-vinod can you rebase and remove the assets changes? Thanks

Reply:
Done